libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -418,6 +418,22 @@ public void setContent(CharSequence text) {
         mAztecReady = true;
     }
 
+    /*
+    * TODO: REMOVE THIS ONCE AZTEC COMPLETELY REPLACES THE VISUAL EDITOR IN WPANDROID APP
+     */
+    private String removeVisualEditorProgressTag(String originalText) {
+        // this regex picks any <progress> tags and any opening <span> tags for image containers
+        // as produced by the Visual Editor. Note that we don't care about closing </span> tags
+        // as the AztecParser takes care of that, and it would be very difficult to accomplish with a
+        // regex (and using a proper XML crawler would be particularly overkill)
+        if (originalText != null && originalText.contains("<progress")) {
+            String regex = "<progress.*?><\\/progress>|<span id=\"img_container.*?>";
```

Comment:
I think it's possible for a user to type `<span id="img_container">`, which would be removed by this regex. How about we make it a little more precise with `<progress.*?><\/progress>|<span id="img_container.*? class="img_container" contenteditable="false">`?

Reply:
👍  addressed in 3e826e8